walkthroughs/week-1-web-development/examples/stanley/pom.xml

```diff
@@ -12,7 +12,7 @@
     <maven.compiler.source>11</maven.compiler.source>
     <maven.compiler.target>11</maven.compiler.target>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <jetty.version>11.0.0</jetty.version>
+    <jetty.version>9.4.31.v20200723</jetty.version>
```

Comment:
The pom changes probably shouldn't be part of this change?

Reply:
You should have argued about this per https://sites.google.com/sps-program.com/students/technical-resources/jetty-9